configure.ac

```diff
@@ -594,6 +594,20 @@ AS_IF([test "$enable_ndb" = yes],[
 ])
 AM_CONDITIONAL([NDB], [test "$enable_ndb" = yes])
 
+#=================
+# Process --enable-bdb-ro
+AC_ARG_ENABLE([bdb-ro], [AS_HELP_STRING([--enable-bdb-ro (EXPERIMENTAL)],[enable the read-only Berkeley DB code])],
+[case "$enable_bdb_ro" in
+yes|no) ;;
+*) AC_MSG_ERROR([invalid argument to --enable-bdb-ro])
+  ;;
+esac],
+[enable_bdb_ro=no])
+AS_IF([test "$enable_bdb_ro" = yes],[
+  AC_DEFINE(WITH_BDB_RO, 1, [Build with read-only Berkeley DB])
+])
+AM_CONDITIONAL([BDB_RO], [test "$enable_bdb_ro" = yes])
+
```

Comment:
Please enable this in CI (ie ci/Dockerfile)

On a related note to self/whoever gets to it first: for whatever reason ndb is not enabled there either, and the others are only implicitly enabled.